src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -24,6 +24,8 @@
     private String delim = DEFAULT_DELIM;
     private String quotes = "always";
     private boolean useTypes = false;
+    private String formatGephi;
+    private Object caption;
```

Comment:
```suggestion
    private String[] caption;
```
e cambiare tutto di conseguenza, quindi il metodo `convertCaption` metterlo qui dentro